docs/proposals/config_data.md

```diff
@@ -0,0 +1,316 @@
+# Generic Configuration Object
+
+## Abstract
+
+This proposal proposes a generic configuration object named `ConfigData` stored inside `etcd` that
```

Comment:
OK but should we create another issue to track it and have discussion around this or its not going to be on the roadmap for lets say k8s 1.2